website/src/data/packages.tsx

```diff
@@ -32,83 +33,131 @@ const Users: Package[] = [
+    tags: ['monitoring'],
+  },
+  {
+    name: 'Velero',
     shortDescription: 'Backup and migrate Kubernetes applications and their persistent volumes',
-    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/44efa073-fd5f-47b8-9ae5-95abb33a47e7',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/943fef9d-629c-4a93-87a7-45193c822fb2',
     websiteUrl: 'https://velero.io/',
     sourceUrl: 'https://github.com/vmware-tanzu/velero',
     tags: ['planned', 'backup'],
   },
   {
-    name: "CloudNativePG",
+    name: 'CloudNativePG',
     shortDescription: 'CloudNativePG covers the full lifecycle of a PostgreSQL database cluster',
-    iconUrl: 'https://avatars.githubusercontent.com/u/100373852?s=150',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/09bc9370-f919-4e04-9429-79dfdb1d12c4',
     websiteUrl: 'https://cloudnative-pg.io/',
     sourceUrl: 'https://github.com/cloudnative-pg/cloudnative-pg',
-    tags: ['planned', 'database'],
+    tags: ['database'],
   },
   {
-    name: "MariaDB Operator",
+    name: 'MariaDB Operator',
     shortDescription: 'Run and operate MariaDB in a cloud native way',
-    iconUrl: 'https://avatars.githubusercontent.com/u/127887858?s=150',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/12c90857-6a3c-416c-bfac-69a26e212240',
     websiteUrl: 'https://github.com/mariadb-operator/mariadb-operator',
     sourceUrl: 'https://github.com/mariadb-operator/mariadb-operator',
     tags: ['planned', 'database'],
   },
+  {
+    name: 'RabbitMQ Operator',
+    shortDescription: 'RabbitMQ Cluster Kubernetes Operator',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/adfe6b1b-625c-4344-aecb-416cd7fccea7',
+    websiteUrl: 'https://www.rabbitmq.com/kubernetes/operator/operator-overview',
+    sourceUrl: 'https://github.com/rabbitmq/cluster-operator',
+    tags: ['planned', 'messaging'],
+  },
+  {
+    name: 'robusta',
+    shortDescription: 'Kubernetes observability and automation, with an awesome Prometheus integration',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/d51cd20b-c769-402e-9cc6-e4e6db33c083',
+    websiteUrl: 'https://home.robusta.dev/',
+    sourceUrl: 'https://github.com/robusta-dev/robusta',
+    tags: ['planned', 'monitoring', 'visualization'],
+  },
+  {
+    name: 'Litmus',
+    shortDescription: 'Litmus helps SREs and developers practice chaos engineering in a Cloud-native way.',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/7e791499-5677-476b-8cd4-5172ac235e5a',
+    websiteUrl: 'https://litmuschaos.io/',
+    sourceUrl: 'https://github.com/litmuschaos/litmus',
+    tags: ['planned', 'security'],
+  },
+  {
+    name: 'Kubeflow',
+    shortDescription: 'RabbitMQ Cluster Kubernetes Operator',
+    iconUrl: 'https://github.com/glasskube/glasskube/assets/3041752/821e6f42-472d-453e-8fb2-c0295c9a17f6)',
```

Comment:
This logo doesn't load for me. Maybe because there is an extra `)` at the end which the others don't have?